### DISCUSSION TITLE
Write CR tag TITLE based on filename

### DISCUSSION THREAD

**Discussion by protofolius** [comictagger/comictagger#505] [Q&A] [answered]
I have changed all my filenames to format 

{series} #{issue} {title} ({year})

When importing these files into comictagger , it succesfully sets 3 fields : Series , Issue and Year .
However , I would like it to also populate the Title field .

Is this somehow possible ?

**Comment by lordwelch** [comment, marked as answer]
You need to enable the "Complicated Parser"
via the CLI
```plain
  --complicated-parser, --no-complicated-parser
                        Enables the new parser which tries to extract more information from filenames (default: False)
```
or in the GUI
![Screenshot_20230901_104043](https://github.com/comictagger/comictagger/assets/7547075/87e105b1-3477-4a05-9359-3ecf38268295)
